src/stores/style.store.ts

```diff
@@ -22,9 +22,16 @@ export const useStyleStore = defineStore(
       new Map()
     )
     const isExpertStyleActive: ShallowRef<boolean> = shallowRef(false)
-    const styleId: ShallowRef<String | null> = shallowRef(null)
+    const styleSerial: ShallowRef<String | null> = shallowRef(null)
     const appliedStyle: ShallowRef<StyleSpecification | undefined> =
       shallowRef()
+    const registerUrls: ShallowRef<Map<string, string>> = shallowRef(
```

Comment:
Don't use stores to configure env. variables, move this into a configurable service instead if needed. By the way, @tkohr has already handled the case, see: https://github.com/Geoportail-Luxembourg/luxembourg-geoportail/commit/e5d151ad5565377653b55df5698399091cce046a